jujugui/static/gui/src/app/views/state.js

```diff
@@ -275,13 +271,6 @@ YUI.add('juju-app-state', function(Y) {
         if (id) {
           id = id.replace(/~charmers\//, '');
         }
-        // Setup the search status and filters based on metadata.search
-        if (!window.flags || !window.flags.react) {
-          if (Y.Lang.isValue(metadata.search)) {
-            search = true;
-            this.filter.update(metadata.search);
```

Comment:
I don't think that `this.filter` (and by extension the class on l780) is used any more, along with the `_addQueryState` method now that the changes on l345 are the default